test/test_helpers/data_migrations.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+# Test your data migrations as follows.
+#
+# require 'test_helper'
+#
+# module DataMigrations
+#   class MyDataMigrationTest < DataMigrationTest
+#     test 'does something' do
+#       migration = MyDataMigration.new
+#       assert_something do
+#         migration.up
+#       end
+#     end
+#   end
+# end
+
+module DataMigrations
+  class DataMigrationTest < ActiveSupport::TestCase ; end
+end
+
+RailsDataMigrations::LogEntry.create_table unless ActiveRecord::Base.connection.table_exists? RailsDataMigrations::LogEntry.table_name
+
+Dir[Rails.root.join *%w[db data_migrations *.rb]].each { |file| require file }
```

Comment:
TBH, we don't _need_ this file.
- Loading the migrations for testing could be done in https://github.com/3scale/porta/blob/master/test/test_helper.rb
- Making sure the `data_migrations` tables exists too
- `DataMigrations::DataMigrationTest` is not essential

The idea of putting it here was to set the convention somewhere. It's open to suggestions anyways.